hack/qci_registry_pruner.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+
+# https://docs.google.com/document/d/1nuLA2q4eqm4_cIPgRzVGV21IJEcqc4IvkMn_LxvACt4/edit#heading=h.66y4kqbj468a
+# describes the QCI registry and our work to use it as the source of truth for promoted images.
+# Unlike the integrated registry on app.ci, there is no automatic pruning in quay.io, so pruning
```

Comment:
```suggestion
# Unlike the integrated registry on app.ci where we do not promote tags with a timestamp, so pruning in quay.io
```

Reply:
I think the reason that app.ci have not hit any issues over the years without the timestamp tags is that the garbage collection on app.ci is not as aggressive as Quay.io.

Reply:
Improving language in next push.